libs/block-template-custom-element/package.json

```diff
@@ -57,13 +58,23 @@
     "displayName": "Template",
     "examples": [
       {
-        "name": "World"
+        "https://blockprotocol.org/@blockprotocol/types/property-type/name/": "World"
       }
     ],
     "icon": "public/omega.svg",
     "image": "public/block-preview.png",
     "name": "block-template-custom-element",
     "protocol": "0.3",
-    "schema": "https://blockprotocol.org/@blockprotocol/types/entity-type/thing/v/2"
+    "codegen": {
+      "targetRoot": "src/types",
+      "targets": {
+        "thing.ts": [
+          {
+            "versionedUrl": "https://blockprotocol.org/@blockprotocol/types/entity-type/thing/v/2",
```

Comment:
Maybe Type ID is better than versioned URL.

`typeId`?
`typeIdToGenerate`?
`sourceTypeId`?

Reply:
[Opt for Type ID instead of Versioned URL in most places](https://github.com/blockprotocol/blockprotocol/pull/1158/commits/3d7d819963407738be9696e6ee2f390feac841b2)